lib/code-explorer.js

```diff
@@ -0,0 +1,272 @@
+var wasmparser = require('wasmparser');
```

Comment:
If you copied code from wasmcodeexplorer, you can put MIT license header here -- I'm the author.